app/assets/stylesheets/_misc_legacy.sass

```diff
@@ -274,7 +274,6 @@ ul.properties
     font-size: 120%
 
 .autoscroll
-  overflow-x: auto
```

Comment:
The `autoscroll` class is applied to other dom elements as well. As it is called `autoscroll` it should probably not loose the scrolling behaviour. I'd instead opt for removing the `autoscroll` class from the dom element.